DEPS

```diff
@@ -15,6 +15,8 @@ vars = {
   're2_revision': 'c84a140c93352cdabbfb547c531be34515b12228',
 
   'spirv_headers_revision': '50daff941d88609b4d2ad076eae558e727f8e5cd',
+
+  'mimalloc_revision': '51c09e7b6a0ac5feeba998710f00c7dd7aa67bbf',
```

Comment:
I see this is https://github.com/microsoft/mimalloc/releases/tag/v3.0.3

I noticed the Validation Layers haven't been updated, I plan to try and update it in VVL to match this

they literally just released a https://github.com/microsoft/mimalloc/releases/tag/v3.1.5 as well, not sure how often/crucial we should be updating this dependency